groups/bsl/bslstl/bslstl_inplace.t.cpp

```diff
@@ -0,0 +1,198 @@
+// bslstl_inplace.t.cpp                                               -*-C++-*-
+
```

Comment:
The component header always comes first.  We never use double-quotes to include files. 

```suggestion

#include <bslstl_inplace.h>
```